int32 unassigned_id
Bbox[] free_bboxes
Bbox[] unknown_bboxes
